src/components/Home/HomeCarousel/HomeCarousel.css

```diff
@@ -0,0 +1,18 @@
+.home-carousel{
+      background-color:  rgb(10, 15, 20);
+}
+.card-image-container{
+      position: relative;
+      width: 100%;
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      height:95vh;
+      background-color: transparent;
+}
+
+.card-image-container > img{
```

Comment:
can you try to fit the images by their height? And limit their width, so it fully takes up all space.